src/main/java/seedu/address/logic/commands/EditPatientCommand.java

```diff
@@ -74,30 +74,30 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
 
-        Person personToEdit = lastShownList.get(index.getZeroBased());
-        Person editedPerson = createEditedPerson(personToEdit, editPersonDescriptor);
+        Person patientToEdit = lastShownList.get(index.getZeroBased());
+        Person editedPerson = createEditedPerson(patientToEdit, editPatientDescriptor);
```

Comment:
would it be better to rename this to editedPatient

Reply:
renamed

Reply:
Nice LGTM